api-verification/locate.spec.ts

```diff
@@ -1,4 +1,6 @@
+import * as path from 'path'
 import { apiClient, tokenAuthorization } from './api-client'
+import { readJsonFile } from './utils'
```

Comment:
Use https://www.typescriptlang.org/tsconfig/resolveJsonModule.html and import directly from JSON file.